lib/compress.js

```diff
@@ -1731,6 +1731,7 @@ merge(Compressor.prototype, {
                 var val = {};
                 for (var i = 0, len = this.properties.length; i < len; i++) {
                     var prop = this.properties[i];
+                    if (prop instanceof AST_Expansion) return this;
```

Comment:
All very good points, but I'm getting tired of working on this feature. :-)

Could you please merge this PR and make any fixes that you think are required? Otherwise I may have time next week.